config/routes.rb

```diff
@@ -41,8 +41,8 @@
     end
   end
 
-  get "users/:id.json" => 'users#show', defaults: {format: 'json'}
-  get 'users/:id/:username' => 'users#show'
+  get "users/:email.json" => 'users#show', defaults: {format: 'json'}, id:"email"
```

Comment:
make it users#edit

In edit, render json: @orgs , print to see what `@orgs` has.